compiler/testData/diagnostics/tests/extensions/additionalReceivers/outerClass.kt

```diff
@@ -0,0 +1,13 @@
+class Outer {
+    val x: Int = 1
+}
+
+with<Outer>
+class Inner {
+    fun bar() = x
+}
+
+fun f() {
+    <!UNRESOLVED_REFERENCE!>Inner<!>()
+    Outer().Inner()
```

Comment:
Please, add a case with implicit outer receiver

Reply:
Also, a test with generic parameter is appreciated
Like `with<B<T>> class A<T>`